### PR TITLE
Infinispan 8.x fixes - 2.x branch

### PR DESCRIPTION
**Pull request by mcimbora**
- org.infinispan.commons.util.concurrent.jdk8backported.ForkJoinPool no longer present in ISPN 8
-- call @Destroy only once, if annotated method is overriden
- Update log4j2 & jboss logging versions

### CODE REVIEW

File: core/src/main/java/org/radargun/config/InitHelper.java [hunk 1]
```diff
@@ -1,20 +1,46 @@
...
+
    public static void init(Object target) {
+      processAnnotatedMethods(target, Init.class);
+   }
+
+   public static void destroy(Object target) {
+      processAnnotatedMethods(target, Destroy.class);
+   }
+
+   private static void processAnnotatedMethods(Object target, Class annotationClass) {
```
Comment (rvansa) [inline core/src/main/java/org/radargun/config/InitHelper.java]: This is not correct - destroy methods must be called in inverse order than inits!

**Comment by mcimbora** [comment]
Fixed.

**Comment by vjuranek** [comment]
Btw: this needs to be pushed also into master branch

**Comment by rvansa** [comment]
Merged through #271

**Pull request closed without merging**